packages/frontend/editor-ui/src/components/canvas/WorkflowCanvas.vue

```diff
@@ -61,8 +61,8 @@ onNodesInitialized(() => {
 	}
 });
 
-const mappedNodesDebounced = debouncedRef(mappedNodes, 200, { maxWait: 50 });
-const mappedConnectionsDebounced = debouncedRef(mappedConnections, 200, { maxWait: 50 });
+const mappedNodesThrottled = throttledRef(mappedNodes, 200, { maxWait: 50 });
+const mappedConnectionsThrottled = throttledRef(mappedConnections, 200, { maxWait: 50 });
```

Comment:
The third parameters is no longer valid
```suggestion
const mappedNodesThrottled = throttledRef(mappedNodes, 200);
const mappedConnectionsThrottled = throttledRef(mappedConnections, 200);
```

Reply:
Thanks, didn't realize.